serverless-templates-gcp-application-python/lab-2/README.md

```diff
@@ -0,0 +1,155 @@
+# Lab 2: Configuration and Application setup for Google Cloud Serverless Function
+
+## Configurations
+In many cases, different stacks for a single project will need differing values.
+The key-value pairs for any given stack are stored in your project’s stack settings file,which is automatically named **`Pulumi.<stack-name>.yaml`**. For example,
+`Pulumi.dev.yaml`
+
+The values are set via [pulumi config set](https://www.pulumi.com/docs/reference/cli/pulumi_config_set/).
+
+More information on this can be found in [setting and getting configuration values](https://www.pulumi.com/docs/intro/concepts/config/#setting-and-getting-configuration-values)
```

Comment:
```suggestion
More information on this can be found in [setting and getting configuration values](https://www.pulumi.com/docs/intro/concepts/config/#setting-and-getting-configuration-values?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=ced-fall2022-workshops)
```